Framework/Algorithms/src/ReflectometryReductionOneAuto.cpp

```diff
@@ -520,11 +520,48 @@ bool ReflectometryReductionOneAuto::checkGroups() {
   }
   return false;
 }
+/**
+ * Sum over transmission group workspaces to produce one
+ * workspace.
+ * @param transGroup : The transmission group to be processed
+ * @return A workspace pointer containing the sum of transmission workspaces.
+ */
+Mantid::API::Workspace_sptr
+ReflectometryReductionOneAuto::sumOverTransmissionGroup(
+    WorkspaceGroup_sptr transGroup) {
+  // Handle transmission runs
+  auto transmissionRunSum = transGroup->getItem(0);
+  MatrixWorkspace_sptr total;
+  for (size_t item = 1; item < transGroup->size(); item++) {
```

Comment:
++item See (here)[http://stackoverflow.com/questions/1392820/why-use-i-instead-of-i-in-cases-where-the-value-is-not-used-anywhere-else-in]  for example